scripts/auto_tick.sh

```diff
@@ -29,21 +29,36 @@ set -e
 source ~/mc/etc/profile.d/conda.sh
 conda env list
 conda activate botenv
+# clone graph repo to get cached graph
+git clone --depth 1 https://github.com/nsls-ii-forge/auto-tick-graph.git
+cp names.txt ./auto-tick-graph/
+cd ./auto-tick-graph/
 # list of packages to be updated (in graph)
 echo "=================="
 cat names.txt
 echo "=================="
-# clean current directory of bot files
-auto-tick clean -y
 # create graph with node_attrs/* and graph.json
 time graph-utils make -o nsls-ii-forge -c -f names.txt -m 10
 # update graph with new versions from their sources (see versions/*)
 time graph-utils update
+# push changes to graph repo
+git add .
+git commit -m "Update graph (auto-tick-bot)"
```

Comment:
Maybe worth to add timestamp of the execution to the commit message to make it a bit more unique?

Something like https://github.com/NSLS-II-SIX/six-utils/blob/499162f8ef0bb36b547dc94bb46674a378f78d01/commit-and-push.sh#L15.

If there is access to the Job number (or some other kind of ID/UID), that would be great to add as well.

Reply:
I guess it will be `BUILD_BUILDNUMBER=20200813.11` env variable (see https://dev.azure.com/nsls2forge/nsls2forge/_build/results?buildId=3767&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5caf77c8-9b10-50ef-b5c7-ca89c63e1c86&l=96).